### ISSUE TITLE
Quicktree textbox should announce selected amount

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode#258617]
Testing #258361

<img width="610" height="297" alt="Image" src="https://github.com/user-attachments/assets/5e8076f1-247b-4c83-a3a2-26b29f374d00" />

It doesn't announce 64 selected
